Sources/HarborJRPC/HJRPCRequestProtocol.swift

```diff
@@ -20,14 +20,13 @@ public protocol HJRPCRequestProtocol {
 }
 
 public extension HJRPCRequestProtocol {
-    func request() async -> HJRPCResponse<Model> {
+    func request() async -> HJRPCResponse<Model> where Model: Sendable {
```

Comment:
I think if you declare Model as Codable && Sendable in the protocol. You will not need **where Model: Sendable**.